### ISSUE TITLE
Selection is not retained on block-quoting

### ISSUE THREAD

**Issue by jodator** [ckeditor/ckeditor5-block-quote#33]
1. Select part of text in paragraph and in adjacent blockquote
2. Execute blockquote command

The selection is truncated to the selection that was already in a blockquote.

Running a list command on the same selection retains it:

![peek 2019-02-21 16-48](https://user-images.githubusercontent.com/247363/53182054-00ea6180-35f9-11e9-917b-ccaeb10ca37a.gif)

**Comment by scofalik** [comment]
Yup this is a known bug. Due to how we handle `wrap` (no longer through `WrapOperation` but `InsertOperation` + `MoveOperation`) we needed to make some compromises in live ranges transformations. Simply saying: not an easy fix, probably won't get to that in foreseeable future.
